lifecycle-operator/apis/lifecycle/v1alpha3/keptnworkload_test.go

```diff
@@ -21,26 +21,6 @@ func TestKeptnWorkload(t *testing.T) {
 		},
 	}
 
-	workloadInstanceName := workload.GetWorkloadInstanceName()
-	require.Equal(t, "workload-version", workloadInstanceName)
-
-	workloadInstance := workload.GenerateWorkloadInstance("prev", map[string]string{})
-	require.Equal(t, KeptnWorkloadInstance{
-		ObjectMeta: metav1.ObjectMeta{
-			Annotations: map[string]string{},
-			Name:        "workload-version",
-			Namespace:   "namespace",
-		},
-		Spec: KeptnWorkloadInstanceSpec{
-			KeptnWorkloadSpec: KeptnWorkloadSpec{
-				Version: "version",
-				AppName: "app",
-			},
-			WorkloadName:    "workload",
-			PreviousVersion: "prev",
-		},
-	}, workloadInstance)
```

Comment:
we are losing this test, but with your argument https://github.com/keptn/lifecycle-toolkit/pull/2255/files#r1354907489, I am in favor of this. This piece of code has been quite stable.